src/styles/main.scss

```
@@ -0,0 +1,147 @@
+$body-font: roboto, sans-serif;
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$white: #fff;
+$concrete-color: #f3f3f3;
+
+body {
+  margin: 0;
+  font-family: $body-font;
+}
```

You can hvae those in index/main file after you import card and stars components
```suggestion
$body-font: Roboto, sans-serif;
$main-accent-color: #060b35;
$secondary-color: #616070;
$blue-accent-color: #00acdc;
$white: #fff;
$concrete-color: #f3f3f3;

body {
  margin: 0;
  font-family: $body-font;
}
```